lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,6 +57,15 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
+    if not isinstance(word_frequencies, dict):
+        return None
+    pair_dict = {}
+    for word in word_frequencies:
+        for i in range(len(word) - 1):
+            if not (word[i], word[i + 1]) in pair_dict:
+                pair_dict[(word[i], word[i + 1])] = 0
+            pair_dict[(word[i], word[i + 1])] += 1 * word_frequencies[word]
```

Comment:
better multiply to `0`

Reply:
simplified this function a bit in hopes that it is what you meant